Bottom: 6f71f27b766d20357551de85122e926729ed80dd
Top:    40bcfc842a59ca6438136be2fcf8c323bd211ac3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:13:31 +0100

Refresh of create-new-function

---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index 80cbde8..2fdb78c 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -613,7 +613,7 @@ gcode_do_export (HID_Attr_Val * options)
             PCB->LayerGroups.Entries[i][0] : i;
           is_bottom =
             (GetLayerGroupNumberByNumber (idx) ==
-             GetLayerGroupNumberByNumber (BOTTOM_SIDE)) ? 1 : 0;
+             GetLayerGroupNumberBySide (BOTTOM_SIDE)) ? 1 : 0;
           save_drill = is_bottom; /* save drills for one layer only */
           gcode_start_png ();
           hid_save_and_show_layer_ons (save_ons);
